sidebars.js

```diff
@@ -984,6 +984,9 @@ module.exports = {
     {
       "Zeebe Gateway": ["self-managed/zeebe-gateway-deployment/zeebe-gateway"],
     },
+    {
+      "Web Modeler (Beta)": ["self-managed/web-modeler/installation"],
```

Comment:
Heads up that due to the fact that we now have separate versions for Optimize than the rest of the docs, changes to sidebars.js require parallel changes to optimize_sidebars.js...at least for now. I'll be updating some documentation soon to reflect this, and I'll share in the #ask-optimize channel. 

But please don't worry about making any optimize_sidebars changes for this PR -- I'll take care of that for you this afternoon, before we do the 8.1 docs release.

Reply:
Thanks for the heads-up and for taking care of the Optimize changes, @pepopowitz! I'm not yet very familiar with how things work in this repository 😊.